pkgs/applications/networking/cluster/kubernetes/default.nix

```diff
@@ -16,16 +16,16 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "kubernetes-${version}";
-  version = "1.9.7";
+  version = "1.10.3";
 
   src = fetchFromGitHub {
     owner = "kubernetes";
     repo = "kubernetes";
     rev = "v${version}";
-    sha256 = "1dykh48c6bvypg51mlxjdyrggpjq597mjj83xgj1pfadsy6pp9bh";
+    sha256 = "1la9cdf5a67kg72xn4bn5mib1caiv5vxsjmnxqsmx0m7vhbv5i4n";
   };
 
-  # go > 1.10 should be fixed by https://github.com/kubernetes/kubernetes/pull/60373
+  # go > 1.10 should be fixed by https://github.com/kubernetes/kubernetes/pull/60597
```

Comment:
This looks like it's already merged, but I think we should follow whichever version kube is built with upstream, which appears to be 1.9.3. Is there any place where this information is located that we can link to instead? Maybe the changelog?

Reply:
According to https://github.com/kubernetes/community/blob/master/contributors/devel/development.md, minimum Golang version required for 1.10 is v1.9.1. And the official version states in External Dependencies for kubernetes 1.10 is v1.9.3: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md

Reply:
Let's link to the External Dependencies instead then.